test/unit/behaviours/tenancy-start-postcode-check.test.js

```diff
@@ -59,60 +59,66 @@ describe('apps/behaviours/tenancy-start-postcode-check', () => {
     });
   });
   describe('saveValues()', () => {
-    let callback;
 
     beforeEach(() => {
-      callback = sinon.stub();
-      sinon.stub(Base.prototype, 'saveValues');
+      sinon.stub(Base.prototype, 'saveValues').yields();
```

Comment:
learner question for myself really, why is a yields needed?  is it because it'sn async function?